enums/uncefactIrradianceUnits.json

```diff
@@ -1,8 +1,9 @@
 {
-    "description": "UN/CEFACT surface irradiance measurement units. In order: Watts w/m2, lumens/m2 (lux).",
+    "description": "UN/CEFACT surface irradiance measurement units. In order: Watts w/m2, lumens/m2 (lux), Joules J/m2.",
     "type": "string",
     "enum": [
         "D54",
-        "B60"
+        "B60",
+        "B13"
```

Comment:
Is it acceptable to have units of different dimensions in the same enum? Watts/m2, Lumens/m2 and Joules/m2 can't be directly converted between them (unlike the area/length/mass enums etc)

Reply:
We have done the same in the case where product is specified - simply having a list of acceptable UN-CEFACT units even if there is not straightforward conversion between them (bushels, kilograms, tubes...).